Documentation/platforms/arm/imxrt/boards/imxrt1020-evk/index.rst

```diff
@@ -0,0 +1,99 @@
+===============
+i.MX RT1020 EVK
+===============
+
+`i.MX RT1020 EVK <https://www.nxp.com/design/development-boards/i-mx-evaluation-and-development-boards/i-mx-rt1020-evaluation-kit:MIMXRT1020-EVK>`_
+is an evaluation kit by NXP company. This kit uses the i.MX RT1020 crossover MCU in LQFP144 package with ARM Cortex M7 core.
+
+Features
+========
+
+- Processor
+    - MIMXRT1021DAG5A processor
+- Memory
+    - 256 Mb SDRAM memory
+    - 64 Mb QSPI Flash
+    - TF socket for SD card
+- Display and Audio
+    - Audio CODEC
+    - 4-pole audio headphone jack
+    - External speaker connection
+    - Microphone
+- Connectivity
+    - Micro USB host and OTG connectors
+    - Ethernet (10/100T) connector
+    - CAN transceivers
+    - Arduino® interface
+
+Serial Console
+==============
+
+The EVK default console is on LPUART1, which is multiplexed onto
+the debug port (either OpenSDA or SEGGER JLink).
+
+It runs at 115200,n,8,1.
+
+LEDs and Buttons
+================
+
+LEDs
+----
+
+There is one user accessible LED status indicator located on the 1020-EVK,
+USERLED.  The function of the LEDs include:
+
+- D3: Power (Green) & Overpower (Red)
+- D5: User LED (Green) GPIO_AD_B0_05
+- D15: RST LED (Red)
```

Comment:
Can you convert this to a table? otherwise formatting won't look right. same thing for similar itemizations on other documents.

Reply:
Sure thing.

Reply:
@v01d Could you take a look at it? I think we can merge it now.